core/src/com/smeanox/games/sg002/view/GameView.java

```diff
@@ -239,6 +255,9 @@ public void render(SpriteBatch spriteBatch, Player activePlayer) {
 				//render GameObjects
 				gameObject = gameWorld.getWorldGameObject(x, y);
 				if (gameObject != null) {
+					spriteBatch.setColor(((float)gameObject.getHp() / gameObject.getGameObjectType().getDefaultHP() > 0.4) ? Color.GREEN : Color.RED);
```

Comment:
Put Colors in Consts.java instead of hard coding them here. It's easier to change things if they are placed together.